packages/nx-playwright/README.md

```diff
@@ -1,24 +1,26 @@
-# nx-playwright
+# @mands/nx-playwright
 
-NX Plugin for Playwright end-to-end test in nx monorepo using the native Playwright runner.
+An [Nx plugin](https://nx.dev/packages/nx-plugin) to add support to an Nx monorepo for
+Playwright testing using a native runner.
 
 ## Usage
 
-`cd` into your NX monorepo.
+Replacing the placeholders `<MONOREPO-PATH>` and `<APP-NAME>`:
 
-`yarn add @mands/nx-playwright -D`
+```sh
+cd <MONOREPO-PATH>
+yarn add -D @mands/nx-playwright
+yarn nx generate @mands/nx-playwright:project <APP-NAME>-e2e --project <APP-NAME>
+yarn playwright install --with-deps
+yarn nx e2e <APP-NAME>-e2e
+```
 
-`yarn nx generate @mands/nx-playwright:project name-of-the-app-e2e --project name-of-the-app`
+## Testing this plugin locally
 
-`yarn playwright install --with-deps`
+Create a new Nx workspace containing one application. Then run, in the root of this repo:
 
-`yarn nx e2e name-of-the-app-e2e`
+```sh
+LOCAL_TEST_WORKSPACE=<path-to-workspace> ./local-test-nx-playwright.sh
+```
 
-## Testing locally
-
-> **Warning** when using the `--cleanup` option, this script will undo any local changes on your target NX workspace
-
-- Create new NX workspace
-- Create a new application
-- Make sure `LOCAL_TEST_WORKSPACE` points to your newly created NX workspace
-- Run `./local-test.sh` (optionally use `--cleanup` to revert changes after the script is run)
+⚠️ Optionally, use the flag `--cleanup` to revert any changes made by the script. However, this will undo **all** local changes made to your target Nx workspace.
```

Comment:
```suggestion
⚠️ Optionally, use the flag `--cleanup` to revert any changes made by the script to your target Nx workspace. However, this will undo **all** local changes made to your target Nx workspace.
```

Reply:
Clarified